roles/chrony/tasks/main.yml

```diff
@@ -17,3 +19,9 @@
     backup: true
   notify: Restart chrony
   
+- name: Restarting chrony
```

Comment:
As discussed on Slack, the task should just notify the handler and do nothing else. Something like this:
```
  - name: Restart chrony
    command: /bin/true
    notify: Restart chrony
```

But test it, I didn't try it.